src/In.ProjectEKA.HipService/Discovery/Ranker/ExactNameRanker.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace In.ProjectEKA.HipService.Discovery.Ranker
+{
+    using HipLibrary.Patient.Model;
+    using Matcher;
+    using static RankBuilder;
+    using static MetaBuilder;
+
+    public class ExactNameRanker : IRanker<Patient>
+    {
+        public PatientWithRank<Patient> Rank(Patient patient, string name)
+        {
+            return (ExactNameMatcher.IsMatch(patient.Name, name)) switch
```

Comment:
Codacy found an issue: [Replace this 'switch' expression with a ternary conditional operator to increase readability.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5933330)